lib/device.js

```diff
@@ -251,6 +256,169 @@ const util = require('util'),
+                    const parseDirPath = path.dirname(options.outputPath),
+                        parseBase = path.parse(options.outputPath).base;
+
+                    let parseExt = path.parse(options.outputPath).ext;
+                    parseExt = parseExt.split('.').pop();
+
+                    log.info("device#capture#_makeCaptureOption():" + " dir name : " + parseDirPath 
+                        + " ,file name : " + parseBase +" ,inputFormat : " +  parseExt);
+
+                    // if given path has extension
+                    if (parseBase && parseExt) {
+                        if (parseExt === "png" || parseExt === "bmp" || parseExt === "jpg") {
+                            captureFormat = parseExt.toUpperCase();
+                        } else {
+                            return setImmediate(next, errHndl.changeErrMsg("INVALID_CAPTURE_FORMAT"));
+                        }
+
+                        captureFileName = parseBase;
+                        destinationPath = path.resolve(parseDirPath);
+                    } else if (parseBase && (parseExt === "")) {
+                        // it's directory path. Use it as destination dir Path
+                        captureFileName = _createDateFileName();
+                        destinationPath = path.resolve(options.outputPath);
+                    }
+                }
+
+                options.captureFormat = captureFormat;
+                options.captureFileName = captureFileName;
+                options.captureDirPath = "/tmp/capture",
+                options.sourcePath = path.join(options.captureDirPath, captureFileName);
+                options.destinationPath = destinationPath;
+                options.ignore = true;
+                options.silent = true;
+                
+                next();
+            }
+
+            function _createDateFileName() {
+                // Capture file name generate : targetname_displayX_yyyymmddhhmmss.png"
+                function __getDate(){
+                    const curDate = new Date();
+                    const dateFormat = __pad(curDate.getFullYear(), 2)
+                        +  __pad((curDate.getMonth()+1), 2)
+                        + __pad(curDate.getDate(), 2)
+                        + __pad(curDate.getHours(), 2)
+                        + __pad(curDate.getMinutes(), 2)
+                        + __pad(curDate.getSeconds(), 2);
+                    return dateFormat;
+                }
+
+                function __pad(number, length) {
+                    let str = '' + number;
+                    while (str.length < length) {
+                        str = '0' + str;
+                    }
+                    return str;
+                }
+
+                const fileName = options.session.target.name + "_" + "display" + options.display+ "_" + __getDate() + ".png";
+                return fileName;
+            }
+
+            function _captureFile(next) {
```

Comment:
how about captureScreen?

Reply:
_captureFile is better, cause already captureScreen () is defined in device.js

Reply:
captureScreenFile();

Reply:
Done.